pkg/restapi/vc/operation/operations_test.go

```diff
@@ -1573,3 +1709,34 @@ func (m *mockVCStatusManager) UpdateVCStatus(v *verifiable.Credential, profile *
 func (m *mockVCStatusManager) GetCSL(id string) (*cslstatus.CSL, error) {
 	return m.getCSLValue, m.getCSLErr
 }
+
+// FailOnSecondOpenMockProvider is a mock provider that returns an err the second time the OpenStore method is called.
+type FailOnSecondOpenMockProvider struct {
```

Comment:
Interestingly enough, I actually have a case where I needed it to fail the second time...  (I remember I asked about this hypothetical situation awhile back @sandrask, but I didn't think I'd actually see it so soon)